core/state_transition.go

```diff
@@ -206,6 +218,14 @@ func (st *StateTransition) preCheck() error {
 				st.msg.From().Hex(), msgNonce, stNonce)
 		}
 	}
+	// Make sure the transaction feeCap is greater than the block's baseFee.
+	if st.evm.ChainConfig().IsAleut(st.evm.Context.BlockNumber) {
+		if st.feeCap.Cmp(st.evm.Context.BaseFee) < 0 {
```

Comment:
This leads to a nil pointer panic when running `eth_estimateGas`, for example if I try:

`> eth.fillTransaction({"from":"0xfe3b557e8fb62b89f4916b721be55ceb828dbd73", "to": "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73"})`

Presumably because `CallArgs.ToMessage` isn't yet Aleut aware so it builds a `StateTransition` with a nil `feeCap`.

Reply:
Oh interesting, nice catch. I will fix.

Reply:
I just realized that for the crash to happen you need my other WIP changes to allow fillTransaction to generate EIP-1559 txs. I'll try to get that code to a reviewable state on Monday.